README.md

```diff
@@ -19,19 +19,29 @@ The library supports the following Java environments:
 
 The following related projects are built on the Google HTTP Client Library for Java:
 
-- [Google OAuth Client Library for Java](https://github.com/googleapis/google-oauth-java-client),
-for the OAuth 2.0 and OAuth 1.0a authorization standards.
-- [Google APIs Client Library for Java](https://github.com/googleapis/google-api-java-client), for
-access to Google APIs.
+- [Google OAuth Client Library for Java][google-oauth-client], for the OAuth 2.0 and OAuth 1.0a
+authorization standards.
+- [Google APIs Client Library for Java][google-api-client], for access to Google APIs.
 
-This is an open-source library, and
-[contributions](https://developers.google.com/api-client-library/java/google-http-java-client/contribute)
-are welcome.
+This is an open-source library, and [contributions][contributions] are welcome.
```

Comment:
should the link be to CONTRIBUTING.md?

Reply:
This link is going to the CONTRIBUTING.md. It's defined at the bottom of the README.md